dev-libs/appstream/metadata.xml

```diff
@@ -7,6 +7,7 @@
 	</maintainer>
 	<use>
 		<flag name="apt">Enables Debians APT support</flag>
+		<flag name="compose">Enable appstream compose</flag>
```

Comment:
That's really a nondescription. What does it do?
Is there no additional dependency? If not, why do we make it optional at all?
Should be committed as a revbump w/ ~arch keywords.

Reply:
so appstream compose isn't required by like 99.999% of users, unless you are using flatpak's or generating appstream data by hand I can't think of a reason you'd need it. In the spirit of gentoo I added a flag since, most people will not need it and hence they should be able to disbale it